### ISSUE TITLE
WARNING: No such namespace: clojure.test

### ISSUE THREAD

**Issue by kommen** [nubank/matcher-combinators#85]
Every time `match?` is used in ClojureScript in an `is` I get this warning:

```
WARNING: No such namespace: clojure.test, could not locate clojure/test.cljs, clojure/test.cljc, or JavaScript source providing "clojure.test"
```

I would guess that this is the cause:

https://github.com/nubank/matcher-combinators/blob/638b7c8dd0ceb881f4edcb38676c3b60ad0f6a10/src/cljc/matcher_combinators/cljs_test.cljc#L32

**Comment by philomates** [comment]
Thanks for reporting @kommen 
I wasn't able to reproduce this with a test but I fixed the namespace usage you mentioned and released it under `1.2.2`.

Please let me know if this didn't fix it.

**Comment by kommen** [comment]
Thanks @philomates, this fixed the issue for us!
